cmd/application_scale.go

```diff
@@ -0,0 +1,62 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/civo/civogo"
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+	"github.com/spf13/cobra"
+)
+
+var processType string
+var processCount int
+
+var appScaleCmd = &cobra.Command{
+	Use:     "scale",
+	Aliases: []string{"change", "modify", "upgrade"},
+	Example: "civo app scale APP-NAME --process-type=web --process-count=3",
```

Comment:
Can we make it possible to do this? 
```suggestion
	Example: "civo app scale APP-NAME web=3 worker=5",
```

Reply:
I don't think. How will we go about so many args? It'll be complex like `arg(process_type)=arg(process_count)`. Also, we support 4 process types as you mentioned which means a series of such args. Is it possible?

Reply:
You can access all args by doing a `os.Args` . Then you can run a for loop with len(os.Args). Each arg should be in the format of `string=string`, so you can use strings.Split(). After that, you have to validate the process type.

Reply:
When you say each arg will be of format `string=string`, the APP-NAME arg doesn't satisy that?

Reply:
strings.Split will give us the slice like so - e.g [web 3], how do we proceed then? How do we set the process count?

Reply:
You'll have to first create an empty slice with ProcInfo : `[]ProcInfo`
Once you have the slice[web, 3], you can append it to the ProcInfo slice . 
Something like this:
`procInfo := append(procInfo, ProcessInfo{ name: stringSlice[0], count: stringSlice[1] } `